docs_versioned_docs/version-ros2humble/components/yaml/attachments/a200/bumper.mdx

```diff
@@ -0,0 +1,36 @@
+<table>
+    <tr>
```

Comment:
@tonybaltovski  consider removing these indents, since the next <td> cell cannot be indented.